lib/workflows/claim_substantiation/nodes/generate_docx_output.py

```diff
@@ -11,6 +17,80 @@
 logger = logging.getLogger(__name__)
 
 
+def get_severity_icon(severity: SeverityEnum) -> str:
+    """Get icon for a severity level."""
+    match severity:
+        case SeverityEnum.HIGH:
+            return "⚠️"
+        case SeverityEnum.MEDIUM:
+            return "⚡"
+        case SeverityEnum.LOW:
+            return "ℹ️"
+        case _:
+            return "📝"
+
+
+def issue_to_comment(
+    issue: DocumentIssue,
+    chunk_content_map: Dict[int, str],
+) -> Optional[DocxComment]:
+    """Convert a DocumentIssue to a DocxComment."""
+    if issue.chunk_index is None:
+        return None
+
+    chunk_content = chunk_content_map.get(issue.chunk_index, "")
+    if not chunk_content:
+        return None
+
+    icon = get_severity_icon(issue.severity)
+    return DocxComment(
+        chunk_index=issue.chunk_index,
+        text=chunk_content,
+        comment_text=f"{icon} {issue.title}\n{issue.description}",
+    )
+
+
+def build_citation_suggestion_comments(
+    state: ClaimSubstantiatorState,
+) -> List[DocxComment]:
```

Comment:
I think you can include this in the rank_issues, so we have parity between the docx comments and the UI comments

Reply:
Nice, I agree, I'll set it to low for now.